src/digitalmarketplace/components/header/template.njk

```diff
@@ -70,7 +70,7 @@
 {{
   govukHeader({
     'productName': productName,
-    'productUrl': '/',
+    'homepageUrl': '/',
```

Comment:
Good catch, I was wondering why the tests didn't fail. The reason is that if `homepageUrl` is not set then it defaults to `/`.